DominicGBauer/system-design-dashboard
# feat: add favicon, routing, excel download, state management and enhance table

## Scope
Add 
1. routing
2. table selection updates
3. excel download
4. favicon
5. Add vuex to handle state management

## Work Done
1. Added routing and added views and routes
2. Added excel download 
3. Enhanced table
4. Added vuex to handle state management
5. Updated favicon to use AIFMRM logo
6. Updated navbar and highlights bar to look more aligned and neater and display it on every page
7. Updated tests
8.  Removed PieCharts component as it's no longer needed

## Steps to Test
1. Run `npm i`
2. Run `npm run start`
3.  Make sure backend is connect with the latest backend develop
4. Make sure routing works by clicking on routes
5. Make sure Table functionality works by playing with it 

**NOTE: ALSI share is giving an issue in calcs because it has a null value which is messing up the calculations and needs to be fixed so ignore check ALSI index selection for now!**

## Screenshot
Demonstrated changes this morning in stand up
